packages/remix-dev/config.ts

```diff
@@ -500,23 +500,9 @@ export async function resolveEntryFiles({
   } else if (userEntryServerFile) {
     entryServerFile = userEntryServerFile;
   } else {
-    let serverRuntime = deps["@react-router/deno"]
-      ? "deno"
-      : deps["@react-router/cloudflare"]
-      ? "cloudflare"
-      : deps["@react-router/node"]
-      ? "node"
-      : undefined;
-
-    if (!serverRuntime) {
-      let serverRuntimes = [
-        "@react-router/deno",
-        "@react-router/cloudflare",
-        "@react-router/node",
-      ];
-      let formattedList = disjunctionListFormat.format(serverRuntimes);
+    if (!deps["@react-router/node"]) {
```

Comment:
This is now the only built-in runtime package so we can simplify the code a lot.